tests/Feature/InstallTest.php

```diff
@@ -49,7 +50,7 @@ public function testInstall(): void
 		 * Clearing things up. We could do an Artisan migrate but this is more efficient.
 		 */
 
-		// The order is important: referring tables must be deleted first, referred tables last
+		Schema::disableForeignKeyConstraints();
```

Comment:
Why this? How is this relevant for the PR?

Reply:
I was trying an optimization to avoid having to disable one of the foreign key below.